types/react-dnd-test-backend/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "quotemark": [true, "double"]
```

Comment:
For new packages, this file should only `"extends": "dtslint/dt.json"`

Reply:
Done!